distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -581,14 +581,41 @@ type Vector a
        Arguments:
        - function: A function that takes an element in the vector and returns
          some transformation of that element.
+       - on_problems: Specifies how to handle any problems that arise in
+         `function`.
+
+       ? Problem Handling
+
+         The result of Errors thrown when executing the function depend on
+         `on_problems`:
+         - Problem_Behavior.Report_Error: The error is thrown.
+         - Problem_Behavior.Report_Warning: The result is `Nothing`, with the
+           error attached as a warning. Currently unimplemented.
+         - Problem_Behavior.Ignore: The result is `Nothing`, and the error is
+           ignored.
+
+         Errors that are thrown when executing the supplied function are wrapped
+         in `Map_Error`, which contains the index at which the error occurred.
+
+         In the `Problem_Behavior.Report_Warning` case, only `MAX_MAP_WARNINGS`
+         warnings are attached to result values. After that, the warnings are
+         dropped, but a count of the additional warnings is attached in an
+         `Additional_Warnings` warning.
```

Comment:
The changes on the Array docs above need to be reflected in Vector docs as well.

Reply:
Done